### DISCUSSION TITLE
natlib 与 biblatex 编译出的参考文献不一致

### DISCUSSION THREAD

**Discussion by liziwl** [tuna/thuthesis#675] [Q&A] [answered]
natlib：
![image](https://user-images.githubusercontent.com/23000702/120068698-a207d280-c0b4-11eb-97a6-b529c3e9cce6.png)

biblatex：
![image](https://user-images.githubusercontent.com/23000702/120068665-72f16100-c0b4-11eb-8e6b-7469160493fa.png)

在2个连续引用时，是否应该改成逗号而不是短横线？三个及以上的连续参考项才需要短横线压缩。

**Comment by hushidong** [comment, marked as answer]
更新一下，biblatex-gb7714-2015就一致了。

**Reply by zepinglee** [reply]
感谢帮忙！

**Comment by hushidong** [comment]
国标的示例是2篇就压缩了，所以现在都改成1-2了，以前版本，是1,2

**Comment by liziwl** [comment]
好的，谢谢回答！
